### PR TITLE
Add release notes for 0.253

### PR DESCRIPTION
**Pull request by bhhari**
# Missing Release Notes
## Abhisek Gautam Saikia
- [x] https://github.com/prestodb/presto/pull/15992 Support shutting down coordinator using /info/state endpoint (Merged by: Timothy Meehan)

## Eran Amar
- [x] https://github.com/prestodb/presto/pull/15812 Eran/binomial functions (Merged by: Rongrong Zhong)

## Reetika Agrawal
- [x] https://github.com/prestodb/presto/pull/15886 Adding schema access rules to file based system access control (Merged by: Rebecca Schlussel)

## Zhongting Hu
- [x] https://github.com/prestodb/presto/pull/15736 Hive Meta Store impersonation access (Merged by: Andrii Rosa)

## vaishnavibatni
- [x] https://github.com/prestodb/presto/pull/16021 Revert "Do not create local projection for constant in PlanRemoteProj… (Merged by: Maria Basmanova)

# Extracted Release Notes
- #15932 (Author: Vic Zhang): Disable empty bucket file for temporary table
  - Set default value for configuration property `hive.create-empty-bucket-files-for-temporary-table` to `false`.
- #16000 (Author: Naveen Kumar Mahadevuni): Fix S3 directory detection based on ContentType header
  - Fix S3 table creation error when the directory location specified was created from AWS console.
- #16026 (Author: frankobe): Fix cumulativeUserMemory initial calculation & use double for cumulativeUserMemory consistently
  - Fix amplified `Cumulative User Memory` metrics calculation.
- #16036 (Author: Vic Zhang): Improve memory usage in TableFinishOperator
  - Track system memory used by column statistics in TableFinishOperator.
- #16036 (Author: Vic Zhang): Improve memory usage in TableFinishOperator
  - Track system memory used by column statistics in TableFinishOperator.
- #16036 (Author: Vic Zhang): Improve memory usage in TableFinishOperator
  - Track system memory used by column statistics in TableFinishOperator.
- #16043 (Author: Dongliang Chen): Update airlift to version 102
  - Fix the version name displayed in the UI.

# All Commits
- ff6484ada42e1bc4a25de89a4de209833d7fc8e5 Track system memory in TableFinishOperator (Vic Zhang)
- 6c161c1baa59c9ba3705872b8f2f64021e2a440b Allow statisitcs page reset in LifespanAndStageState (Vic Zhang)
- eaafec018443836081df6082374ef473aec7c94d Track column statistics only in recoverable mode (Vic Zhang)
- dbe485dbaeace0bf1ecd3f2a498ff79522016024 Fix NPE in HiveSpiltManager when there are upper case letter in the partition columns (Beinan Wang)
- cd6ac2d24371d118edeaee927fe6c45d512b1228 Use HIVE_COMPATIBLE function for ORC format (Vic Zhang)
- 8c3542e0ab7aca9728ee0dfae645d9d3999fe74f Use PRESTO_NATIVE for hive unsupported type (Vic Zhang)
- 72a7afc4eba8c3302cb156e1900a8cdfb4e0f72d Revert "Fix dynamic pruning for null keys in hive partition" (Ke Wang)
- f98c25e277f2010e95d8bca4b23c1cd391eb2a8e Bump Alluxio client version (Bin Fan)
- 5f4a45e46173f76201c58b4ee1469a4fce08b773 Use getInt to avoid boxing in LookupJoinPageBuilder (James Petty)
- a87fe3d21587d8a10c9430ddfa9f6ff9b724c8e6 Fix Size Calculation in MapBlock (Arunachalam Thirupathi)
- 0925a963981048896ace46a49082d475230c642d Make HashTable computation Lazy in MapBlockBuilder (Arunachalam Thirupathi)
- 55f99279442b165e3a4086a219ace7c94c74237e Update the Sample output (Zhongting Hu)
- 07999c4215bc3d78b4680a8acaccb04302195939 Add Split Detail into maxActiveSplits endpoint (Zhongting Hu)
- 7ee814c65ba17abf2c071bd8c12d068cd6628bf4 Move TableDescriptionSupplier to separate guice module (Yang Yang)
- bfcd99ee9fd7173dc99628aa6d5fd5dac5bd8054 Add binomial distribution functions (Eran Amar)
- 9543424f8aa44c6bfa9d76ee6432b22fcd15ac73 Disable empty bucket file for temporary table (Vic Zhang)
- 6058064301b7d8b4dab22329343ad0c308c50cab Fix loading config properties for temp storage (Vic Zhang)
- 3a28daf6c473e25f5fdf8bfdb735f5befea7abfd Avoid unnecessary variable for ConstantExpression in PlanRemoteProjections (Rongrong Zhong)
- c2c7b783d0e95f934aff8a1a2a03f939c4b21aa8 Move audience-annotations depencency to provided scope (Tim Meehan)
- 7a17553656594e632c27240d7fe128337fa73549 Update airlift to version 102 (Dongliang Chen)
- eda728ee36df8734d28a912969fcbfc47ead96e4 Supporting adding additional modules to `DistributedQueryRunner` (Mayank Garg)
- 30cd883136f01b004101d50a4169aabab8ed47b1 Add more options to ColumnWriterOptions (Arunachalam Thirupathi)
- 969c84712d2ef6be00305b9cbcfb68dc7cc65ddc Rename CompressionParameters to ColumnWriterOptions (Arunachalam Thirupathi)
- 1e086105fdb0808316e72d15928d83eabe81be34 Adding session config: limit_larger_for_segment for Pinot connector (Xiang Fu)
- 35cfbd05304b9ca41ec6f5d1642a4b4a1f1889d6 Materialized view partition stitching for Hive (Rohit Jain)
- 7e32c30f73d40180f4d3580ff0dc0206f879091c Enable query and node heartbeats to resource manager (Tim Meehan)
- ea52ee04c4b55f538ab87d9129c8624ce796e1e0 Fix cumulativeUserMemory initial calculation & use double for cumulativeUserMemory consistently (frankobe)
- b29aaa3f57cb413d34655e766bf31829f0375c47 Support Hive Metastore access with impersonation (Zhongting Hu)
- d3c3445dca6b5a5f853260b374a32e1b9a64138b Enumerate and serialize splits iteratively (Andrii Rosa)
- 1c2f7e057ce7c10e4c7e0318ae023a357c4cdc3a Use zstd streams with no finalizer (Andrii Rosa)
- 6b5434f91158f48271822258c0b10c5197ccecd6 Revert "Do not create local projection for constant in PlanRemoteProjections" (vaishnavibatni)
- b80c82ed3f78a69596b0a522129a9edf3b4b1fcf Fix S3 directory detection based on ContentType header (Naveen Kumar Mahadevuni)
- ea8e5f4b83f128f20afeb1d79599af4f8caac286 Adding schema access rules to file based system access control (Reetika Agrawal)
- 4710def6239d1fa7e8df247b06f03dd5c14e46c0 Add ResourceManagerClusterStateProvider (Tim Meehan)
- 6b367064979b40a4d8ab5d163abcfe1fd1b6e613 Support shutting down coordinator using /info/state endpoint (Abhisek Gautam Saikia)
- fc25cc49041b92465bdd8b1a12cc8724985656b2 Fix formatted query with PREPARE statement (Shixuan Fan)

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
+* Fix the version name displayed in the UI.
+* Track system memory used by column statistics in TableFinishOperator.
+* Track system memory used by column statistics in TableFinishOperator.
+* Track system memory used by column statistics in TableFinishOperator.
```
Comment (ajaygeorge) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:17]: please dedup

**Comment by bhhari** [comment]
@mayankgarg1990 @ajaygeorge please review, polished the release notes

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
+* Fix the version name displayed in the UI.
+* Add tracking for system memory used by column statistics in TableFinishOperator.
+* Add support for shutting down coordinator via `/v1/info/state` endpoint.
+* Add :func:`binomial_cdf` and :func:`inverse_binomial_cdf` functions.
+
+Security Changes
+----------------
+* Add functionality to set schema access rules in file-based system access control
```
Comment (ajaygeorge) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:21]: end with period.

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
+* Fix the version name displayed in the UI.
+* Add tracking for system memory used by column statistics in TableFinishOperator.
+* Add support for shutting down coordinator via `/v1/info/state` endpoint.
```
Comment (ajaygeorge) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:16]: I think we need double quotes here. Can you try generating the docs offline and see if this highlighted.

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* Add :func:`binomial_cdf` and :func:`inverse_binomial_cdf` functions.
+
+Security Changes
+----------------
+* Add functionality to set schema access rules in file-based system access control
+
+Hive Changes
+____________
+* Fix S3 table creation error when the specified directory location is created from AWS console.
```
Comment (ajaygeorge) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:25]: Is this is a Hive change?

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
+* Fix the version name displayed in the UI.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:14]: Not really sure this should go in at all.

File: presto-docs/src/main/sphinx/release/release-0.253.rst (hunk 4 above)
Comment (tdcmeehan) [reply presto-docs/src/main/sphinx/release/release-0.253.rst:25]: It's a bug fix, and typically S3 metadata is stored in Hive, so seems to belong here.

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+**Highlights**
+==============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metrics calculation.
```
Comment (tdcmeehan) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:13]:
```suggestion
* Fix amplified `Cumulative User Memory` metric in web UI.
```

File: presto-docs/src/main/sphinx/release/release-0.253.rst (hunk 4 above)
Comment (bhhari) [reply presto-docs/src/main/sphinx/release/release-0.253.rst:25]: keeping it as per @tdcmeehan suggestion

**Review by ajaygeorge — APPROVED** [review]
Please check if there any features we can highlight and add it to the Highlights section.
If not please remove it.

Rest looking good.
Please generate the docs offline and test whether everything is rendered as expected before merging.

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+Release 0.253
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metric in web UI.
+* Add tracking for system memory used by column statistics in TableFinishOperator.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:11]: `TableFinishOperator` in double back quotes

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 8]
```diff
@@ -0,0 +1,28 @@
+=============
+Release 0.253
+=============
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix amplified `Cumulative User Memory` metric in web UI.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:10]: Does single back quote do anything - as per my understanding it needs to be double back quotes

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+* Add tracking for system memory used by column statistics in TableFinishOperator.
+* Add support for shutting down coordinator via ``/v1/info/state`` endpoint.
+* Add :func:`binomial_cdf` and :func:`inverse_binomial_cdf` functions.
+
+Security Changes
+________________
+* Add functionality to set schema access rules in file-based system access control.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:17]: `Add support for schema access control in file based system access control`

File: presto-docs/src/main/sphinx/release/release-0.253.rst [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+
+Hive Changes
+____________
+* Fix S3 table creation error when the specified directory location is created from AWS console.
+* Add Hive metastore impersonation access. This can be enabled with the ``hive.metastore-impersonation-enabled`` configuration property.
+* Add Load balancing of multiple Hive metastore instances. This can be enabled with the ``hive.metastore.load-balancing-enabled`` configuration property.
```
Comment (mayankgarg1990) [inline presto-docs/src/main/sphinx/release/release-0.253.rst:23]: `load` instead of `Load`

**Review by mayankgarg1990 — APPROVED** [review]
just a few nits

File: presto-docs/src/main/sphinx/release/release-0.253.rst (hunk 8 above)
Comment (bhhari) [reply presto-docs/src/main/sphinx/release/release-0.253.rst:10]: single back tick make it italic

Comment (rschlussel) [reply presto-docs/src/main/sphinx/release/release-0.253.rst:10]: I think single back ticks are for links. Italics should be between single asterisks.

**Review by highker — APPROVED** [review]

**Pull request merged**